stm32cube/stm32h7xx/README

```diff
@@ -68,4 +68,9 @@ Patch List:
      Impacted files:
       stm32cube/stm32h7xx/drivers/src/stm32h7xx_hal_sdio.c
 
+   *Fix an issue in the STM32H7 HAL OSPI by correcting the DMA source and destination
+      addresses in the HAL_MDMA_Start_IT() call.
+      Impacted files:
+      stm32cube/stm32h7xx/drivers/src/stm32h7xx_hal_ospi.c
```

Comment:
Is there an internal issue associated to be sure this is fixed directly in HAL ?